srcpkgs/python3-cchardet/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-cchardet'
+pkgname=python3-cchardet
```

Comment:
This doesn't seem strictly necessary from what I'm looking:

https://github.com/kovidgoyal/calibre/blob/5d126692f8c35a9a246795ea4757757f01b1e35f/src/calibre/ebooks/chardet.py#L126

It just defaults to a configured encoding if the module can't be imported. But since you already made the new package, I think it's fine to add directly as a dependency, no need to install only for checks.